src/modules/vtol_att_control/standard.cpp

```diff
@@ -335,22 +285,18 @@ void Standard::update_transition_state()
 		q_sp.copyTo(_v_att_sp->q_d);
 		_v_att_sp->q_d_valid = true;
 
-		hrt_abstime btrans_start;
-		btrans_start = _vtol_schedule.transition_start + uint64_t(_params_standard.reverse_delay) * 1000000.0f;
 		_pusher_throttle = 0.0f;
 
-		if (hrt_absolute_time() >= btrans_start) {
+		if (time_since_trans_start >= _params_standard.reverse_delay) {
 			// Handle throttle reversal for active breaking
-			float thrscale = (float)hrt_elapsed_time(&btrans_start) / (_params_standard.front_trans_dur *
-					 1000000.0f);
+			float thrscale = (time_since_trans_start - _params_standard.reverse_delay) / (_params_standard.pusher_ramp_dt);
```

Comment:
@RomanBapst 
This was intentionally scaled over the front transition rampup, not the reverse delay.
The logic being that if the motor requires rampup from zero for a front transition it will require the same when going to reverse. This should not be scaled over the reverse delay, this delay is intended for the motor to stop the prop from spinning before attempting to go in reverse.